packages/oraft/oraft.0.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "komamitsu@gmail.com"
+authors: ["Mitsunori Komatsu"]
+homepage: "https://github.com/komamitsu/oraft"
+bug-reports: "https://github.com/komamitsu/oraft/issues"
+dev-repo: "git+https://github.com/komamitsu/oraft.git"
+description: "Raft consensus algorithm implemented in OCaml"
+synopsis: "Raft consensus algorithm implemented in OCaml"
+license: "Apache"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+    "dune" {build}
```

Comment:
Also `dune` should not be tagged as a build dependency and no `dune-project` file is present in your source repository. Could you untag it and add the appropriate version of dune required according to your `dune-project` file?

Reply:
Ah, I happened to add `dune-project` to my global gitignore unexpectedly... Just untagged `dune` and added `dune-project` file.